charts/kubernetes-external-secrets/templates/rbac.yaml

```diff
@@ -25,6 +25,9 @@ rules:
   - apiGroups: ["kubernetes-client.io"]
     resources: ["externalsecrets"]
     verbs: ["get", "watch", "list"]
+  - apiGroups: ["kubernetes-client.io"]
```

Comment:
can we also update the rbac in external-secrets.yml file https://github.com/godaddy/kubernetes-external-secrets/blob/master/external-secrets.yml#L18 with those permissions

Reply:
Fixt !